src/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+function Footer() {
+  return (
+    <footer>
+      <p className="copyright">
+        © 2022 - Made by{' '}
+        <a className="footerLink" href="https://github.com/pinggyu">
```

Comment:
can add target="_blank" rel="noopenner" / rel="noreferrer" to open on a seperate tab so user can still access you're site